src/test/java/seedu/address/logic/parser/buyer/BuyerCommandParserTest.java

```diff
@@ -95,6 +96,11 @@ public void parseCommand_view() throws Exception {
         assertTrue(parser.parseCommand(ViewCommand.COMMAND_WORD, "") instanceof ViewCommand);
     }
 
+    @Test
+    public void parseCommand_clear() throws Exception {
+        assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD, "") instanceof ClearCommand);
```

Comment:
Good test case to check for the bug regarding clearing command only working for buyers